datalad/support/sshconnector.py

```diff
@@ -49,14 +50,18 @@ def __init__(self, ctrl_path, host):
 
         host: str
           host to connect to. This may include the user ( [user@]host )
+
+        port: str
+          port to connect over
         """
         self._runner = None
 
         # TODO: This may actually also contain "user@host".
         #       So, better name instead of 'host'?
         self.host = host
-        self.ctrl_path = ctrl_path
-        self.cmd_prefix = ["ssh", "-S", self.ctrl_path, self.host]
+        self.ctrl_path = ctrl_path + ":" + port if port else ctrl_path
+        self.port = port
+        self.ctrl_options = ["-o", "ControlPath=\'" + self.ctrl_path + "\'"]
```

Comment:
I don't think we need ' and to escape them when creating the option. It is a list, so passed as a list, si arguments are separated etc... or am I wrong?